content/docs/command-reference/remote/add.md

```diff
@@ -233,12 +233,12 @@ $ dvc remote modify myremote oss_endpoint endpoint
 ```
 
 To set key id, key secret and endpoint (or any other OSS parameter), use
-`dvc remote modify`. Example usage is show below. Make sure to use the `--local`
-option to avoid committing your secrets with Git:
+`dvc remote modify` as show below. Use the `--local` option to avoid committing
+your secrets to Git:
 
 ```dvc
-$ dvc remote modify myremote --local oss_key_id 'mykey'
-$ dvc remote modify myremote --local oss_key_secret 'mysecret'
+$ dvc remote modif --localy myremote oss_key_id 'mykey'
```

Comment:
typo?

Reply:
Thanks
```suggestion
$ dvc remote modify --localy myremote oss_key_id 'mykey'
```

Reply:
Hmmm for some reason I can no longer commit suggestions on Github :(

Reply:
Now the option exists again! Gh bug fixed I suppose. 🤷